src/vue/pages/dashboard/Dashboard.InfoWidget.vue

```diff
@@ -0,0 +1,429 @@
+<template>
```

Comment:
looks like it copy-pasted. we can ommit this file and wait until https://github.com/tokend/web-client/pull/21 merged to reuse the tx-history template from there